lib/utils.ts

```diff
@@ -24,5 +24,6 @@ export const getNameString = (authors: string[] | string) => {
     return authors.join(' and ');
   }
 
-  return authors.join(', ');
+  var last = authors.pop();
```

Comment:
Should it be `const last` here @nelsonmestevao ?

Reply:
Yes 💪

Reply:
Done! :)